docs/connections.md

```diff
@@ -39,10 +39,10 @@ Mongoose lets you start using your models immediately, without waiting for
 mongoose to establish a connection to MongoDB.
 
 ```javascript
-mongoose.connect('mongodb://127.0.0.1:27017/myapp');
+await mongoose.connect('mongodb://127.0.0.1:27017/myapp');
```

Comment:
i dont think this is appropriate for this section of the docs

Reply:
I agree with @hasezoey . Because this section is about buffering, we shouldn't have `await` here.